lib/solidus_dev_support/templates/extension/gitignore

```diff
@@ -14,3 +14,4 @@ pkg
 *.swp
 spec/dummy
 spec/examples.txt
+sandbox
```

Comment:
Can you add a slash at the beginning to tell git we wanna ignore just the toplevel sandbox (vs. `sandbox` inside `bin/`)?

Reply:
oooh good catch! Will do! Thanks!